chia/cmds/keys_funcs.py

```diff
@@ -21,7 +21,7 @@ def generate_and_print():
     mnemonic = generate_mnemonic()
     print("Generating private key. Mnemonic (24 secret words):")
     print(mnemonic)
-    print('Note that this key has not been added to the keychain. Run chia keys add_seed -m "[MNEMONICS]" to add')
+    print('Note that this key has not been added to the keychain. Run chia keys add "[MNEMONICS]" to add')
```

Comment:
I dont think we pass the 24 words on the command line anymore. Rather it prompts the user after chia keys add

Reply:
Good point, I actually thought about that and confirmed the same and meant to strip the [MNEMONICS] part but must have forgotten... I'll update the PR